litmus-portal/backend/tools/self-deployer/deployer.go

```diff
@@ -75,44 +70,55 @@ func RegSelfCluster(server, pid string) (string, error) {
 }
 
 //Deploy function deploys the Manifest received by the RegSelfCluster function
-func Deploy(file string, kubeconfig *string) error {
-	resp, err := http.Get(file)
-	if err != nil {
-		log.Print(err.Error())
-		return err
-	}
-	defer resp.Body.Close()
-	body, err := ioutil.ReadAll(resp.Body)
-	if err != nil {
-		log.Print(err.Error())
-		return err
-	}
-	log.Print(string(body))
-	response, err := k8s.CreateDeployment(body, kubeconfig)
-	if err != nil {
-		log.Println(err.Error())
+func Deploy(file string) error {
+	cmd := exec.Command("kubectl", "apply", "-f", file)
+	if output, err := cmd.Output(); err != nil {
+		log.Print("XX FAILED TO DEPLOY SUBSCRIBER COMPONENTS")
+		log.Print("ERROR : ", err.Error())
 		return err
+	} else {
+		log.Printf("\n%s", output)
```

Comment:
This print `output` not required

Reply:
Output is required for debugging purpose.